src/main/java/io/swagger/oas/inflector/utils/DefaultSpecFilter.java

```diff
@@ -6,39 +6,40 @@
 import io.swagger.oas.models.Operation;
 import io.swagger.oas.models.media.Schema;
 import io.swagger.oas.models.parameters.Parameter;
+import java.util.Optional;
 
 
 import java.util.List;
 import java.util.Map;
 
 public class DefaultSpecFilter extends AbstractSpecFilter {
 
-    /*@Override
-    public boolean isOperationAllowed(Operation operation, ApiDescription api, Map<String, List<String>> params, Map<String, String> cookies, Map<String, List<String>> headers) {
+    @Override
+    public Optional<Operation> filterOperation(Operation operation, ApiDescription api, Map<String, List<String>> params, Map<String, String> cookies, Map<String, List<String>> headers) {
         if(operation.getExtensions() != null && operation.getExtensions().containsKey(Constants.X_INFLECTOR_HIDDEN)) {
-            return false;
+            return Optional.of(operation);
```

Comment:
I guess you want to return `Optional.empty()` here

Reply:
Yes. I've change it. Thanks!